frontend/src/components/Sidebar/Sidebar.tsx

```diff
@@ -39,6 +39,10 @@ const Sidebar: FunctionComponent = ({ children }) => {
             </div>
           </Link>
         ))}
+        <div className={styles.logout}>
+          <img alt="dummy_profile" src="https://sincerelyvictoriat.com/wp-content/uploads/2019/11/round-profile-picture-1.png" />
+          <Link to="">Logout</Link>
```

Comment:
we actually have a logout function, it can be accessed by calling useContext (https://reactjs.org/docs/hooks-reference.html#usecontext) on the context in `context/auth.ts`.

for actually showing profile pics, i dont think it should interfere with the main profile pic PR thats still in progress so you can implement that too. it'll require some modification of the AuthManager and the context i mentioned before since we need to get the user's id, so if you dont have experience with contexts feel free to dm me and i can explain further. for the dummy pfp you can just do a gray circle too